webcomponents/tool/src/main/frontend/packages/sakai-conversations/src/SakaiAddTopic.js

```diff
@@ -716,43 +720,48 @@ export class SakaiAddTopic extends SakaiElement {
         </div>
         ` : nothing}
 
-
         <div id="post-options-block" class="add-topic-block">
           <div id="post-options-label" class="add-topic-label">${this._i18n.post_options}</div>
           ${this.canPin ? html`
           <div>
-            <input type="checkbox" id="pinned-checkbox"
+            <input type="checkbox"
+              id="pinned-checkbox"
+              aria-labelledby="pinned-checkbox-label"
```

Comment:
Does this need to be in i18n? also there are some input element above that does not have aria-label

Reply:
No, because it is declaring which element labels this input. I can't see any other input tags without either an aria-label or arai-labelledby. Which input?